### PR TITLE
Matrix Addition

### PR DESCRIPTION
**Pull request by AmrutaDhage20**

### CODE REVIEW

File: PracticePrograms/src/SixthJune/READme.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Java Program to add two matrices :
+Input :
+firstMatrix =  {{1, 3, 4}, {2,4,3},{3,4,5}}
+secondMatrix =  {{1,3,4},{2,4,3},{1,2,4}}
+
+Output :
+sumOfMatrices = {{2,6,8},{4,8,6},{4,6,9}}
```
Comment (waghmaresuyog) [inline PracticePrograms/src/SixthJune/READme.md:7]: java file is missing

**Review by waghmaresuyog — CHANGES_REQUESTED** [review]

File: PracticePrograms/src/SixthJune/MatrixAddition.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+public class MatrixAddition {
+
+    public static void main(String args[]) {
+        int firstMatrix[][] = {{1, 3, 4}, {2, 4, 3}, {3, 4, 5}};
+        int secondMatrix[][] = {{1, 3, 4}, {2, 4, 3}, {1, 2, 4}};
+
+        int sumOfMatrices[][] = new int[3][3];
+
+        for (int i = 0; i < 3; i++) {
+            for (int j = 0; j < 3; j++) {
```
Comment (waghmaresuyog) [inline PracticePrograms/src/SixthJune/MatrixAddition.java]: give meaningful name to variable

File: PracticePrograms/src/SixthJune/MatrixAddition.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+public class MatrixAddition {
+
+    public static void main(String args[]) {
+        int firstMatrix[][] = {{1, 3, 4}, {2, 4, 3}, {3, 4, 5}};
+        int secondMatrix[][] = {{1, 3, 4}, {2, 4, 3}, {1, 2, 4}};
+
+        int sumOfMatrices[][] = new int[3][3];
+
+        for (int i = 0; i < 3; i++) {
```
Comment (waghmaresuyog) [inline PracticePrograms/src/SixthJune/MatrixAddition.java]: give meaningful name to variable

File: PracticePrograms/src/SixthJune/MatrixAddition.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+            for (int j = 0; j < 3; j++) {
+                sumOfMatrices[i][j] = firstMatrix[i][j] + secondMatrix[i][j];  //use - for subtraction
```
Comment (waghmaresuyog) [inline PracticePrograms/src/SixthJune/MatrixAddition.java]: same here

File: PracticePrograms/src/SixthJune/MatrixAddition.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+                System.out.print(sumOfMatrices[i][j] + " ");
```
Comment (waghmaresuyog) [inline PracticePrograms/src/SixthJune/MatrixAddition.java]: same here

File: PracticePrograms/src/SixthJune/MatrixAddition.java (hunk 4 above)
Comment (waghmaresuyog) [reply PracticePrograms/src/SixthJune/MatrixAddition.java]: change the comment as per operation

**Review by waghmaresuyog — CHANGES_REQUESTED** [review]

File: PracticePrograms/src/SixthJune/MatrixAddition.java (hunk 2 above)
Comment (AmrutaDhage20) [reply PracticePrograms/src/SixthJune/MatrixAddition.java]: Done

File: PracticePrograms/src/SixthJune/MatrixAddition.java (hunk 3 above)
Comment (AmrutaDhage20) [reply PracticePrograms/src/SixthJune/MatrixAddition.java]: Done

File: PracticePrograms/src/SixthJune/MatrixAddition.java (hunk 4 above)
Comment (AmrutaDhage20) [reply PracticePrograms/src/SixthJune/MatrixAddition.java]: Done

File: PracticePrograms/src/SixthJune/MatrixAddition.java (hunk 5 above)
Comment (AmrutaDhage20) [reply PracticePrograms/src/SixthJune/MatrixAddition.java]: Done

File: PracticePrograms/src/SixthJune/MatrixAddition.java [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+        int firstMatrix[][] = {{1, 3, 4}, {2, 4, 3}, {3, 4, 5}};
+        int secondMatrix[][] = {{1, 3, 4}, {2, 4, 3}, {1, 2, 4}};
+
+        int sumOfMatrices[][] = new int[3][3];
+
+        for (int rowIndex = 0; rowIndex < 3; rowIndex++) {
+            for (int columnIndex = 0; columnIndex < 3; columnIndex++) {
+                //Addition of matrices
+                sumOfMatrices[rowIndex][columnIndex] = firstMatrix[rowIndex][columnIndex] + secondMatrix[rowIndex][columnIndex];
```
Comment (waghmaresuyog) [inline PracticePrograms/src/SixthJune/MatrixAddition.java:15]: comment is missing

**Review by waghmaresuyog — CHANGES_REQUESTED** [review]

**Review by waghmaresuyog — APPROVED** [review]
Approve

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**